articles/how-to-convert-text-to-emoji-using-python/index.md

```diff
@@ -0,0 +1,155 @@
+# HOW TO CONVERT TEXT TO EMOJI USING PYTHON
+
+![Hero image](/engineering-education/how-to-convert-text-to-emoji-using-python/hero.jpg)
+
+### Introduction
+Emojis are frequently used on social media to communication expressions and emotions. As a programmer, have you ever considered writing code that produces emojis in Python? Well, this article is for you.
+
+In this tutorial, you will learn how to build an emoji converter from text using dictionaries in Python, which store data as key-value pairs. Python is a high-level programming language that is quick to read and execute. It has a vast library that can perform other programming tasks such as changing files and text searching.
+
+It is also a scripting language and is often used to create Web applications, dynamic Web content automation, and Artificial intelligence.
+
+### Table of content
+- Prerequisites
+- Variables
+- Strings and Formatted strings
+- The IF and ELSE statement
+- The Input and print function
+- Dictionaries
+- Text to emoji converter
+- conclusion
+
+### Prerequisites
+As a beginner in Python, you can use your browser to download the most recent version of [Python](https://www.python.org/ftp/python/3.9.2/python-3.9.2-macosx10.9.pkg) and install it on your computer. 
+You will also need to download a python code editor more preferable [PyCharm](https://www.jetbrains.com/pycharm/download/download-thanks.html?platform=mac), which is an integrated development environment for professional developers, and it's also student-oriented, friendly, and easy to use.
+
+### Variables
+Variables are the most fundamental concepts in programming, particularly Python. Variables are labels that signify where you can store data in your computer memory, and they may also contain values. They can store simple values like strings, integers, and Boolean values. Variables should always be in lowercase, preferably.
```

Comment:
This statement is too generic and a bit inaccurate. Variables are fundamental concepts particularly in Python? Introduce variables directly and talk about the various data types Python supports.